Source/Scene/ModelExperimental/PointCloudStylingPipelineStage.js

```diff
@@ -266,26 +266,20 @@ function getVariableSubstitutionMap(propertyAttributes) {
     const propertyAttribute = propertyAttributes[i];
     const properties = propertyAttribute.properties;
     for (const propertyId in properties) {
+      // This will have already been sanitized for GLSL by
```

Comment:
Minor rewording for clarity

```suggestion
      // The property ID was already sanitized for GLSL by PntsLoader
```